draft_1_proposal_instructions.md

```diff
@@ -0,0 +1,144 @@
+In the future, the contents of the "SNS lifecycle" section in ./README.md would
+be replaced with the content in this file (minus this preface, ofc).
+
+_This section assumes that you have successfully completed the instructions in
+the [Bootstrapping a testing environment via Docker] section above._
+
+This section explains how to use the aforementioned testing environment to
+perform SNS decentralization of your own dapp running in the local `dfx` server
+test environment.
+
+The commands mentioned here are to be run from within the Docker container.
+
+'run_basic_scenario.sh` also performs these steps. Therefore, reading that
```

Comment:
We should have a similar script for the new 1-proposal approach (and test it on CI).